srv/components/misc_pkgs/openldap/config/base.sls

```diff
@@ -34,7 +36,9 @@ Remove existing file:
 
 Configure OpenLDAP - Schema:
   cmd.run:
-    - name: ldapadd -x -D "cn=admin,cn=config" -w {{ salt['lyveutil.decrypt']('openldap', pillar['openldap']['admin']['secret']) }} -f /opt/seagate/cortx/provisioner/generated_configs/ldap/cn\=\{1\}s3user.ldif -H ldapi:///
+    - name: ldapadd -x -D "cn=admin,cn=config" -w $password -f /opt/seagate/cortx/provisioner/generated_configs/ldap/cn\=\{1\}s3user.ldif -H ldapi:///
+    - env:
+      - password: {{ salt['lyveutil.decrypt']('openldap', pillar['openldap']['admin']['secret']) }}
```

Comment:
can we avoid that copy-paste: it's not only error prone but also not optimal since each time decryption will happen